src/PodCreateModal.jsx

```diff
@@ -66,6 +66,22 @@ export const PodCreateModal = ({ user, systemServiceAvailable, userServiceAvaila
         return createConfig;
     };
 
+    /* Updates a validation object of the whole dynamic list's form (e.g. the whole port-mapping form)
+    *
+    * Arguments
+    *   - key: [publish/volumes/env] - Specifies the validation of which dynamic form of the Image run dialog is being updated
+    *   - value: An array of validation errors of the form. Each item of the array represents a row of the dynamic list.
+    *            Index needs to corellate with a row number
+    */
+    const dynamicListOnValidationChange = (value, key) => {
+        setValidationFailed(prevState => {
+            prevState[key] = value;
+            if (prevState[key].every(a => a === undefined))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1482-20231110-112222-2df94782-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)